plugin-flex-ts-template-v2/src/feature-library/keyboard-shortcuts/custom-components/KeyboardShortcuts/Tabs/CustomKeyboardShortcutsView.tsx

```diff
@@ -0,0 +1,173 @@
+import React, { useEffect, useState } from 'react';
+import {
+  Heading,
+  Paragraph,
+  Card,
+  Tooltip,
+  Text,
+  Stack,
+  Box,
+  Table,
+  THead,
+  Tr,
+  Th,
+  Td,
+  TBody,
+} from '@twilio-paste/core';
+import { WarningIcon } from '@twilio-paste/icons/esm/WarningIcon';
+import { InformationIcon } from '@twilio-paste/icons/esm/InformationIcon';
+
+import EditButton from '../EditButton';
+import KeyCommand from '../KeyCommand';
+import ModalWindow from '../ModalWindow';
+import DeleteButton from '../DeleteButton';
+import { ShortcutsObject } from '../../../types/types';
+import { getCustomShortcuts } from '../../../utils/KeyboardShortcutsUtil';
+
+interface CustomKeyboardShortcutsViewProps {
+  reset: boolean;
+  disableShortcuts: boolean;
+  isThrottleEnabled: boolean;
+  isDeleteShortcutsEnabled: boolean;
+  toasterDeleteNotification: (actionName: string) => void;
+  toasterSuccessNotification: (actionName: string, oldShortcut: string, newShortcut: string) => void;
+}
+
+const CustomKeyboardShortcutsView = ({
+  reset,
+  disableShortcuts,
+  isThrottleEnabled,
+  isDeleteShortcutsEnabled,
+  toasterDeleteNotification,
+  toasterSuccessNotification,
+}: CustomKeyboardShortcutsViewProps) => {
+  const [customShortcuts, setCustomShortcuts] = useState<ShortcutsObject[]>([]);
+  const [shortcutsDeleted, setShortcutsDeleted] = useState<boolean>(false);
+  const [isEditModalOpen, setIsEditModalOpen] = useState<boolean>(false);
+  const [selectedShortcutKey, setSelectedShortcutKey] = useState<string>('');
+  const [selectedActionName, setSelectedActionName] = useState<string>('');
+  const [selectedThrottle, setSelectedThrottle] = useState<number | undefined>(0);
+
+  const openModalHandler = (shortcut: string, actionName: string, throttle: number | undefined): void => {
+    setIsEditModalOpen(true);
+    setSelectedThrottle(throttle);
+    setSelectedShortcutKey(shortcut);
+    setSelectedActionName(actionName);
+  };
+
+  const closeModalHandler = (): void => {
+    setIsEditModalOpen(false);
+  };
+
+  useEffect(() => {
+    setCustomShortcuts(getCustomShortcuts());
+    setShortcutsDeleted(false);
+  }, [reset]);
+
+  useEffect(() => {
+    if (Object.keys(getCustomShortcuts()).length === 0) {
+      setShortcutsDeleted(true);
+    }
+  }, [customShortcuts]);
+
+  return (
+    <>
+      {/* Check if shortcuts have been disabled or deleted  */}
+      {disableShortcuts || shortcutsDeleted ? (
+        <Card>
+          <Heading as="h5" variant="heading50">
+            <Stack orientation="horizontal" spacing="space20">
+              <WarningIcon decorative={false} title="Description of icon" />
```

Comment:
This icon title can be removed and made decorative.

Reply:
Done!